README.md

```diff
@@ -13,6 +13,7 @@ The following implementations are available:
 * `intel-mkl`, which is the one in [Intel MKL],
 * `netlib`, which is the reference one by [Netlib], and
 * `openblas`, which is the one in [OpenBLAS].
+* `r`, which is the one in [R].
```

Comment:
It was a bit off if you look at the whole list. The “and” should have been moved from line 14 to line 15.